client/my-sites/plans/main.jsx

```diff
@@ -70,6 +74,20 @@ var Plans = React.createClass( {
 		);
 	},
 
+	redirectToDefault() {
+		page.redirect( paths.plans( this.props.selectedSite.slug ) );
+	},
+
+	renderNotice() {
```

Comment:
Do we need to use this approach? Couldn't we create a persistent notice when the request finishes, and just redirect to the standard plans page.